ISSUE_TEMPLATE.md

```diff
@@ -0,0 +1,13 @@
+### Issue and Steps to Reproduce
+<!-- Describe your issue and tell us how to reproduce it (include any useful information). -->
+
+### Versions
+
+### Screenshots
+
+#### Expected
+
+#### Actual
+
+### Additional Details
+* Installed packages:
```

Comment:
What do you think people would enter here? I would think that it would be included in the problem description.